src/tools/Makefile.am

```diff
@@ -30,7 +30,8 @@ noinst_HEADERS = util.h fread_to_eof.h \
 	openpgp-tool-helpers.h
 bin_PROGRAMS = opensc-tool opensc-explorer opensc-asn1 \
 	pkcs15-tool pkcs15-crypt pkcs11-tool pkcs11-register \
-	cardos-tool eidenv openpgp-tool iasecc-tool egk-tool goid-tool
+	cardos-tool eidenv openpgp-tool iasecc-tool egk-tool goid-tool \
+	dtrust-tool
```

Comment:
Please, add the tool also to the windows makefile:

https://github.com/OpenSC/OpenSC/blob/master/src/tools/Makefile.mak

Reply:
fixed